torch/csrc/autograd/variable.cpp

```diff
@@ -81,21 +137,23 @@ namespace impl {
       auto diff_view_meta = static_cast<DifferentiableViewMeta*>(get_autograd_meta(self));
 
       // See NOTE [ View + Inplace detection ]
-      if (diff_view_meta->creation_meta != CreationMeta::MULTI_OUTPUT_SAFE) {
+      auto creation_meta = diff_view_meta->get_creation_meta();
```

Comment:
This could have been done ahead of time in a separate PR, where you only introduce the backward ViewInfo and flip everything over to the new API.